### PR TITLE
Restyle dispatch log

### PR DESCRIPTION
**Pull request by benmelz**
Based off #365

Haven't yet ported over coloring for pinned entries (and maybe we don't have to? not sure). Needs javascript implementation from @aflam745

### CODE REVIEW

File: app/assets/javascripts/log_entries.js [hunk 1]
```diff
@@ -1,3 +1,34 @@
 $(document).on('turbolinks:load', function() {
   // andrew writes javascript here
```
Comment (benmelz) [inline app/assets/javascripts/log_entries.js:2]:
```suggestion
```

File: app/assets/javascripts/log_entries.js [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+  $('.log-entry-edit-button').on('click', function (e) {
+    var theClickedElement = $(e.target);
+    var parent = theClickedElement.closest('.log-entry');
+    var edit = parent.find('.log-entry-edit-button');
+    var delet = parent.find('.log-entry-delete-button');
```
Comment (benmelz) [inline app/assets/javascripts/log_entries.js:7]:
```suggestion
    var delete = parent.find('.log-entry-delete-button');
```
nitpick

File: app/assets/javascripts/log_entries.js [hunk 3]
```diff
@@ -1,3 +1,34 @@
...
+    var show = parent.find('.log-entry-show');
+    var form = parent.find('.log-entry-edit');
+    var cancel = parent.find('.log-entry-edit-cancel-button');
+
+    edit.prop('hidden', true);
+    delet.prop('hidden', true);
```
Comment (benmelz) [inline app/assets/javascripts/log_entries.js:13]:
```suggestion
    delete.prop('hidden', true);
```

File: app/assets/javascripts/log_entries.js [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+    show.prop('hidden', true);
+    form.prop('hidden', false);
+    cancel.prop('hidden', false);
+  });
+
+  $('.log-entry-edit-cancel-button').on('click', function (e) {
+    var theClickedElement = $(e.target);
+    var parent = theClickedElement.closest('.log-entry');
+    var edit = parent.find('.log-entry-edit-button');
+    var delet = parent.find('.log-entry-delete-button');
```
Comment (benmelz) [inline app/assets/javascripts/log_entries.js:23]:
```suggestion
    var delete = parent.find('.log-entry-delete-button');
```

File: app/assets/javascripts/log_entries.js [hunk 5]
```diff
@@ -1,3 +1,34 @@
...
+    var show = parent.find('.log-entry-show');
+    var form = parent.find('.log-entry-edit');
+    var cancel = parent.find('.log-entry-edit-cancel-button');
+
+    edit.prop('hidden', false);
+    delet.prop('hidden', false);
```
Comment (benmelz) [inline app/assets/javascripts/log_entries.js:29]:
```suggestion
    delete.prop('hidden', false);
```

**Review by aflam745 — APPROVED** [review]

**Pull request merged**